examples/two-chip.rs

```diff
@@ -218,38 +214,24 @@ impl<F: FieldExt> AddInstructions<F> for AddChip<F> {
                 // but we can only rely on relative offsets inside this region. So we
                 // assign new cells inside the region and constrain them to have the
                 // same values as the inputs.
-                let lhs = region.assign_advice(
-                    || "lhs",
-                    config.advice[0],
-                    0,
-                    || a.value.ok_or(Error::Synthesis),
-                )?;
-                let rhs = region.assign_advice(
-                    || "rhs",
-                    config.advice[1],
-                    0,
-                    || b.value.ok_or(Error::Synthesis),
-                )?;
-                region.constrain_equal(a.cell, lhs)?;
-                region.constrain_equal(b.cell, rhs)?;
+                a.0.copy_advice(|| "lhs", &mut region, config.advice[0], 0)?;
+                b.0.copy_advice(|| "rhs", &mut region, config.advice[1], 0)?;
 
-                // Now we can assign the multiplication result into the output position.
-                let value = a.value.and_then(|a| b.value.map(|b| a + b));
-                let cell = region.assign_advice(
-                    || "lhs * rhs",
-                    config.advice[0],
-                    1,
-                    || value.ok_or(Error::Synthesis),
-                )?;
+                // Now we can assign the addition result into the output position.
```

Comment:
```suggestion
                // Now we can compute the addition result, which is to be
                // assigned into the output position.
```
(The code up to the next comment no longer does the assignment.)